Kunena/Kunena-Forum
# Remove editb2.js and editfa.js from Crypsis template

Pull Request for Issue # .
#### Summary of Changes

Need to remove editb2.js and editfa.js to avoid duplicate code
#### Testing Instructions

Use the setting in Crypsis to choose the topicicontype between font awesome or boostrap 2 and check in quick reply and write a message if all is ok
